packages/core/src/filters/defaultFilter.vert

```diff
@@ -2,25 +2,27 @@ attribute vec2 aVertexPosition;
 
 uniform mat3 projectionMatrix;
 
-varying vec2 vTextureCoord;
```

Comment:
You can't change uniform name, because all custom filters stops work if it has custom vertex shader.
This is no case for loss of compatibility .